ftw/publisher/receiver/browser/configure.zcml

```diff
@@ -18,4 +18,10 @@
         permission="cmf.ReviewPortalContent"
         />
 
+    <browser:page
+        name="get_saved_data"
```

Comment:
Is this name specific enough? I think it could conflict with other things. Should we prefix the name?
(I assume it is called on the site root).